src/main/java/ui/listpanel/ListPanel.java

```diff
@@ -564,11 +564,12 @@ private void markAsRead() {
         }
     }
 
-    private void createNewWatchListMenuItem() {
+    /**
+     * Creates a new watch list panel containing the currently selected issue
+     */
+    private void createNewWatchListPanel() {
         Optional<GuiElement> item = listView.getSelectedItem();
-        if (!item.isPresent()) {
-            return;
-        }
+        assert item.isPresent();
```

Comment:
So it's fine if the issue disappears before this code gets executed, right?

I've been trying this a few times and couldn't get the assert to fail.

Guess the user has to be so unlucky to encounter this error since the update must occur after the user clicks the corresponding menu but before the first line of the method gets executed (i.e. `Optional<GuiElement> item = listView.getSelectedItem();`) :s 

@dariusf any opinion regarding this issue?